### PR TITLE
Fix volume rendering link crash

### LINKED ISSUE #4805: Can't load scene that contains a volumes synced GPU volume rendering
Cannot load a scene that contains a volume rendering that is synced to the volumes module

## Steps to reproduce

1. download MRHead sample data
2. Got to Volume Rendering module
3. Make sure GPU rendering is enabled
3. Select the volume and make visible
4.  In the advanced section, click the synchronize checkbutton
5. (Optional) Change the color table in the Volumes module to verify that it is synced
6.  Save the scene
7.  Close Slicer, reopen, and attempt to load the scene
8. Loading hangs

## Expected behavior
 The scene should load correctly

## Environment
- Slicer version: Slicer-4.11.0-2020-04-01
- Operating system: Windows10

## More details

In the scene file , the ```GPURayCastVolumeRendering``` tag contains the ```followVolumeDisplayNode="1"``` attribute.  Changing this to ```"0"``` to unlink allows the scene to load.  This bug does not occur with GPU rendering

### PR DESCRIPTION
**Pull request by sjh26**
Closes #4805 

Per the Tuesday discussion, I have replaced the asserts with warnings.  I've also added a check on scene importing when updating add volume render nodes.

Did not need to listen for the end of the scene import to update the volume rendering.  The update is also triggered by OnMRMLNodeModified, so it will occur after the scene import anyway.

### CODE REVIEW

File: Modules/Loadable/VolumeRendering/Logic/vtkSlicerVolumeRenderingLogic.cxx [hunk 1]
```diff
@@ -170,7 +173,13 @@ void vtkSlicerVolumeRenderingLogic::AddVolumeRenderingDisplayNode(vtkMRMLVolumeR
   it = this->DisplayNodes.insert(this->DisplayNodes.end(), static_cast<vtkMRMLNode*>(nullptr));
   // .. then set and observe
   vtkSetAndObserveMRMLNodeMacro(*it, node);
-  this->UpdateVolumeRenderingDisplayNode(node);
+
+  //Don't update volume rendering while a scene is being imported
+  if (!this->GetMRMLScene()->IsImporting())
+  {
```
Comment (lassoan) [inline Modules/Loadable/VolumeRendering/Logic/vtkSlicerVolumeRenderingLogic.cxx:179]: indentation

File: Modules/Loadable/VolumeRendering/Logic/vtkSlicerVolumeRenderingLogic.cxx [hunk 2]
```diff
@@ -404,7 +417,13 @@ void vtkSlicerVolumeRenderingLogic::UpdateTranferFunctionRangeFromImage(vtkMRMLV
 void vtkSlicerVolumeRenderingLogic::SetThresholdToVolumeProp(
   double scalarRange[2], double threshold[2], vtkVolumeProperty* volumeProp, bool linearRamp, bool stayUpAtUpperLimit)
 {
-  assert(scalarRange && threshold && volumeProp);
+
+  if (!volumeProp || !scalarRange || !threshold)
+    {
+    vtkWarningMacro("SetThresholdToVolumeProp: Inputs  do not exist.");
```
Comment (lassoan) [inline Modules/Loadable/VolumeRendering/Logic/vtkSlicerVolumeRenderingLogic.cxx:423]: double space

File: Modules/Loadable/VolumeRendering/Logic/vtkSlicerVolumeRenderingLogic.cxx [hunk 3]
```diff
@@ -439,7 +458,11 @@ void vtkSlicerVolumeRenderingLogic::SetThresholdToVolumeProp(
 void vtkSlicerVolumeRenderingLogic::SetWindowLevelToVolumeProp(
   double scalarRange[2], double windowLevel[2], vtkLookupTable* lut, vtkVolumeProperty* volumeProp)
 {
-  assert(scalarRange && windowLevel && volumeProp);
+  if (!volumeProp || !scalarRange || !windowLevel)
+    {
+    vtkWarningMacro("SetWindowLevelToVolumeProp: Inputs  do not exist.");
```
Comment (lassoan) [inline Modules/Loadable/VolumeRendering/Logic/vtkSlicerVolumeRenderingLogic.cxx:463]: double space

**Comment by lassoan** [comment]
Looks good, thank you @sjh26!

**Comment by sjh26** [comment]
Updated as requested

**Pull request merged**